src/main/kotlin/io/github/karmasmp/karma/messenger/NoxesiumMessenger.kt

```diff
@@ -17,6 +17,10 @@ class NoxesiumMessenger : PluginMessageListener {
 
     private fun parseIntByteArray(bytes : ByteArray) : Int {
         // Could quite possibly be destroyed if Daniel changes how information is sent from Noxesium.
-        return bytes[1].toInt()
+        return try {
+            bytes[1].toInt()
+        } catch(e: Exception) {
+            0
```

Comment:
Should probably at least log something rather than fail silently

Reply:
Added logger and dev message.